icepll/icepll.cc

```diff
@@ -25,6 +25,20 @@
 #include <emscripten.h>
 #endif
 
+enum feedback_mode
```

Comment:
This should probably be an `enum struct` so the enum values are properly scoped.